ClimsoftVer4/ClimsoftVer4/ucrHourly.vb

```diff
@@ -242,15 +242,16 @@ Public Class ucrHourly
                 If elemTotal = expectedTotal Then
                     bValueCorrect = True
                 Else
-                    MessageBox.Show("Value in [Total] textbox is different from that calculated by computer! " & elemTotal, "Error in total", MessageBoxButtons.OK, MessageBoxIcon.Error)
-                    ucrInputTotal.SetBackColor(Color.Cyan)
-                    bValueCorrect = False
-                End If
-                bValueCorrect = (elemTotal = expectedTotal)
-                If Not bValueCorrect Then
                     MessageBox.Show("Value in [Total] textbox is different from that calculated by computer! The computed total is " & elemTotal, "Error in total", MessageBoxButtons.OK, MessageBoxIcon.Error)
-                    ucrInputTotal.SetBackColor(Color.Cyan)
+                    ucrInputTotal.SetBackColor(Color.Red)
+                    ucrInputTotal.GetFocus()
+                    bValueCorrect = False
                 End If
+                'bValueCorrect = (elemTotal = expectedTotal)
+                'If Not bValueCorrect Then
+                '    MessageBox.Show("Value in [Total] textbox is different from that calculated by computer! The computed total is " & elemTotal, "Error in total", MessageBoxButtons.OK, MessageBoxIcon.Error)
+                '    ucrInputTotal.SetBackColor(Color.Cyan)
+                'End If
```

Comment:
I thought this was the shorter version of the code you have retained

Reply:
What do you mean?

Reply:
The code you have commented is similar but shorter than the one you have retained.